Snowflake.Data.Tests/IntegrationTests/SFBindTestIT.cs

```diff
@@ -9,511 +9,451 @@
 namespace Snowflake.Data.Tests.IntegrationTests
 {
     using NUnit.Framework;
-    using Snowflake.Data.Client;
-    using Snowflake.Data.Core;
+    using Client;
+    using Core;
     using System.Text;
     using System.Globalization;
     using System.Collections.Generic;
 
-    [TestFixture]    
-    class SFBindTestIT : SFBaseTest
+    [TestFixture]
+    internal class SFBindTestIT : SFBaseTest
     {
         [Test]
         [Ignore("BindTestIT")]
         public void BindTestITDone()
         {
             // Do nothing;
         }
-
+ 
         [Test]
-        public void testArrayBind()
+        public void TestArrayBind()
         {
-            using (IDbConnection conn = new SnowflakeDbConnection())
+            CreateOrReplaceTable(TestName, new []
```

Comment:
Maybe we should have a method to translate TestName into proper tableName?
Currently TestName == tableName, but it could be confusing. I'm thinking of something like:
```
 CreateOrReplaceTable(TestTable(), ...
```

Reply:
Good idea, done that.